spec/linter-spec.js

```diff
@@ -2,20 +2,29 @@
 
 import * as path from 'path';
 
+const travisYml = path.join(__dirname, '.travis.yml');
```

Comment:
This file doesn't exist, since `__dirname` refers to the spec folder. Perhaps you meant `path.join(__dirname, '..', '.travis.yml'`?